cider/exec/template/CompilationOptions.h

```diff
@@ -62,7 +62,7 @@ enum class ExecutorType { Native, Extern, TableFunctions };
 struct ExecutionOptions {
   bool output_columnar_hint;
   bool allow_multifrag;
-  bool just_explain;  // return the generated IR for the first step
+  uint32_t just_explain;  // return the generated IR for the first step
```

Comment:
Maybe ```enum``` is more explicit. Just an advice.

Reply:
yep